### ISSUE TITLE
Player jitters upon reaching target

### ISSUE THREAD

**Issue by supersuade** [rtourgeeunm/Amorphous_Clone#1]
Player sometimes jitters upon reaching target.

Adding a distance check before setting velocity in process could be a solution.

This issue is just so I can practice opening and issue and maybe creating a pull request.
As a side note, I am enjoying breaking your game for once.

**Comment by rtourgeeunm** [comment]
I noticed that too.
I'll have to find a way to set velocity to 0 once the player is within a certain range of the target.
Perhaps if the target is within the player's collision shape?
I've also switched the target selector off and made the player follow the mouse entirely. This will keep the player moving so long as the mouse is moving which will be always if the player wants to stay alive. Still, worth fixing if it's decided that the player's target isn't the mouse down the line.

Glad you're enjoying it. I'll make plenty for you to break.
